omnisafe/configs/off-policy/SACRCBF.yaml

```diff
@@ -0,0 +1,134 @@
+    parallel: 1
+    # total number of steps to train
+    total_steps: 200000
+    # number of evaluate episodes
+    eval_episodes: 1
+  # algorithm configurations
+  algo_cfgs:
+    # number of steps to update the policy
+    steps_per_epoch: 1000
+    # number of steps per sample
+    update_cycle: 1
+    # number of iterations to update the policy
+    update_iters: 1
+    # The size of replay buffer
+    size: 1000000
+    # The size of batch
+    batch_size: 256
+    # normalize reward
+    reward_normalize: False
+    # normalize cost
+    cost_normalize: False
+    # normalize observation
+    obs_normalize: False
+    # max gradient norm
+    max_grad_norm: 40
+    # use critic norm
+    use_critic_norm: False
+    # critic norm coefficient
+    critic_norm_coeff: 0.001
+    # The soft update coefficient
+    polyak: 0.005
+    # The discount factor of GAE
+    gamma: 0.99
+    # Actor perdorm random action before `start_learning_steps` steps
+    start_learning_steps: 5000
+    # The delay step of policy update
+    policy_delay: 1
+    # Whether to use the exploration noise
+    use_exploration_noise: False
+    # The exploration noise
+    exploration_noise: 0.1
+    # The policy noise
+    policy_noise: 0.2
+    # policy_noise_clip
+    policy_noise_clip: 0.5
+    # The value of alpha
+    alpha: 0.2
+    # Whether to use auto alpha
+    auto_alpha: True
+    # use cost
+    use_cost: False
+  # control barrier function configurations
+  cbf_cfgs:
+    # gamma of control barrier certificate.
+    gamma_b: 20
+    # confidence parameter desired
+    k_d: 3.0
+    # environment dynamics coefficient
+    l_p: 0.03
+  # logger configurations
+  logger_cfgs:
+    # use wandb for logging
+    use_wandb: False
+    # wandb project name
+    wandb_project: omnisafe
+    # use tensorboard for logging
+    use_tensorboard: True
+    # save model frequency
+    save_model_freq: 40
+    # save logger path
+    log_dir: "./runs"
+    # save model path
+    window_lens: 10
+  # model configurations
+  model_cfgs:
+    # weight initialization mode
+    weight_initialization_mode: "kaiming_uniform"
+    # actor type
+    actor_type: gaussian_sac
+    # linear learning rate decay
+    linear_lr_decay: False
+    # Configuration of Actor network
+    actor:
+      # Size of hidden layers
+      hidden_sizes: [400, 300]
+      # Activation function
+      activation: relu
+      # The learning rate of Actor network
+      lr: 0.0003
+    # Configuration of Critic network
+    critic:
+      # The number of critic networks
+      num_critics: 2
+      # Size of hidden layers
+      hidden_sizes: [400, 300]
+      # Activation function
+      activation: relu
+      # The learning rate of Critic network
+      lr: 0.0003
+  # Dynamics model configurations
+  dynamics_model_cfgs:
+    # The max number of episodes updating GP models
+    gp_max_episodes: 100
+    # The size of gp model
+    gp_model_size: 2000
+    # Whether to use the action compensator
```

Comment:
can we also unify the case spell in configs? I propose maybe we can consider keeping lowercase except for proper noun.